src/components/view/index.jsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+
+import {
+  FormControl,
+  RadioGroup,
+  FormControlLabel,
+  Radio,
+  FormLabel,
+  Checkbox,
+} from "@mui/material";
+
+const OPTION_TYPE = {
+  single: "single",
+  multi: "multi",
+};
+
+// eslint-disable-next-line react/prop-types
+function MultipleChoiceQuestion({ options }) {
+  return (
+    <FormControl component="form">
+      <FormLabel component="question">What is the capital of France?</FormLabel>
+      {OPTION_TYPE.single !== "single" ? (
```

Comment:
Refactor the `OPTION_TYPE.single !== "single"` condition to directly check the option type without comparing the same value, which currently always results in false. [important]